framework/src/org/checkerframework/framework/type/DefaultTypeHierarchy.java

```diff
@@ -934,6 +943,11 @@ public Boolean visitWildcard_Intersection(
     @Override
     public Boolean visitWildcard_Primitive(
             AnnotatedWildcardType subtype, AnnotatedPrimitiveType supertype, VisitHistory visited) {
+        if (subtype.isUninferredTypeArgument()) {
```

Comment:
Should this situation ever even happen? Shouldn't the primitive be boxed before it is compared to a wildcard?